assignment-day-2/src/sql/queries/truncate_employee_data.sql

```diff
@@ -0,0 +1 @@
+TRUNCATE TABLE raw_employee;
```

Comment:
Since we are deleting the table for both employee and timesheet. How about we add all such queries in a constant file separately and then use it? This can avoid repetition. 
Sample: 
`TRUNCATE_TABLE= "TRUNCATE TABLE  {table_name}"`